sonar-project.properties

```diff
@@ -3,10 +3,10 @@ sonar.projectKey=sap_cloud-commerce-spartacus-storefront
 sonar.projectName=Cloud commerce spartacus storefront
 sonar.projectVersion=0.1.0
 
-sonar.sources=projects/storefrontlib,projects/core,projects/storefrontstyles
+sonar.sources=projects/storefrontlib,projects/cds,projects/core,projects/storefrontstyles
 sonar.exclusions=**/node_modules/**,**/*.spec.ts
 
-sonar.tests=projects/storefrontlib,projects/core,projects/storefrontstyles
+sonar.tests=projects/storefrontlib,projects/cds,projects/core,projects/storefrontstyles
```

Comment:
@hackergil can you please take a look at these changes?

Reply:
it's ok